src/locale/en.json

```diff
@@ -0,0 +1,55 @@
+{
+  "page-title": "COVID-19 Housing Protection Legislation and Housing Justice Action Map",
+  "about-title": "COVID-19 Housing Protection Legislation & Housing Justice Action",
+  "about-map": "About the map:",
+  "about-protections": "Housing Protection Legislation",
+  "about-description": "Where officials have passed (or tenants are working to pass) housing protection legislation during the COVID-19 crisis.",
+  "legislation-form-usa": "Submit housing protection legislation for locations in the United States.",
+  "legislation-form-intl": "Submit housing protection legislation for locations outside of the United States.",
+  "policyStrength": {
```

Comment:
We'll also need a translation in the legend for "expired protections" (this was just updated in master)

```
"expired": "<em>Stripes indicate expired protections</em>"
```

cc @unixjazz

Reply:
This didn't need the `<em>` tags